meson.build

```diff
@@ -276,4 +276,9 @@ if get_option('apps')
 endif
 
 pkg = import('pkgconfig')
-pkg.generate(nanoarrow_lib)
+pkg.generate(libraries : nanoarrow_lib,
+             subdirs : [incdir],
+             version : meson.project_version(),
```

Comment:
You shouldn't need to provide this - the current pkgconfig generates this from the project version